scipy/spatial/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from scipy.spatial.distance import _METRICS_NAMES
+
+
+@pytest.fixture(params=_METRICS_NAMES)
```

Comment:
We should do this once per session only
```suggestion
@pytest.fixture(params=_METRICS_NAMES, scope="session")
```